extensions/JeremyGamer13/tween.js

```diff
@@ -0,0 +1,311 @@

```

Comment:
just save this image as `images/JeremyGamer13/tween.png` and add any needed copyright information to `images/README.md` (follow the existing pattern, should not be hard)